### PR TITLE
Migrate e2e test for Elasticsearch to Go

### LINKED ISSUE #3233: Migrate e2e test for Elasticsearch to Go
### Proposal

Relates to https://github.com/kedacore/keda/issues/2357, https://github.com/kedacore/keda/issues/2737.

### Use-Case

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by aviadlevy**
Signed-off-by: aviadlevy <aviadlevy1@gmail.com>

Added elasticsearch e2e test.  
I needed to change the `ParseCommand` in `helpers.go` in order to support the somewhat complex `exec` commands needed for this tests. let me know if you think of a better solution.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [x] Tests have been added

Fixes #3233

### CODE REVIEW

File: tests/scalers_go/elasticsearch/elasticsearch_test.go [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+
+	elasticsearchDummyDoc = `
+{
+  "@timestamp": "{{.Timestamp}}",
+  "dummy": 1,
+  "dumb": "oOooo"
+}`
+)
+
+var (
```
Comment (zroubalik) [inline tests/scalers_go/elasticsearch/elasticsearch_test.go:283]: Just a nit, could you please move this above templates, so sections of the test are aligned with the example mentioned here: https://github.com/kedacore/keda/tree/main/tests#example-test-lets-say-you-want-to-add-a-test-for-redis

Comment (aviadlevy) [reply tests/scalers_go/elasticsearch/elasticsearch_test.go]: Done

**Comment by zroubalik** [comment]
@aviadlevy could you plese fix the static checks issue?

**Comment by aviadlevy** [comment]
> @aviadlevy could you plese fix the static checks issue?

on it. need

**Comment by aviadlevy** [comment]
> @aviadlevy could you plese fix the static checks issue?

missed https://github.com/kedacore/keda/commit/2790448a057f25b050bf9e25ed4fb72b1d92a377 is merged. 
Rebasing and fixing

**Review by zroubalik — APPROVED** [review]
LGTM, thanks a lot!

**Pull request merged**